src/graphql/operations/user.ts

```diff
@@ -1,13 +1,18 @@
+import { capture } from '@snapshot-labs/snapshot-sentry';
 import db from '../../helpers/mysql';
-import { formatAddress, formatUser } from '../helpers';
 import log from '../../helpers/log';
-import { capture } from '@snapshot-labs/snapshot-sentry';
+import { formatUser, formatAddresses, PublicError } from '../helpers';
 
 export default async function (parent, args) {
-  const id = formatAddress(args.id);
+  const addresses = formatAddresses(
+    [args.id],
+    ['evmAddress', 'starknetAddress']
+  );
+  if (!addresses.length) throw new PublicError('Invalid address');
+
```

Comment:
Feel too confusing, it can't autoformat address.?

Reply:
i think we should let `formatAddresses` take care of types and errors

Reply:
I we throw the error from `formatAddress`, we can not have a detailed error message explaining which field the error is coming from